che-theia-java-extension/src/browser/che-theia-java-contribution.ts

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Can you check what formatting conventions the other Che Theia plugins use, please?

Reply:
Do you mean for the license or in general?